controllers/UserController.js

```diff
@@ -593,6 +593,17 @@ const getAffiliate = async (req, res, next) => {
   next();
 };
 
+const getMinReject = async (req, res, next) => {
+  const value = validators
+    .validate(req.body, validators.user.minRejectSchema, next);
+  if (!value) return;
+
+  const json = await calcVoteValue.getMinReject(value);
+
+  res.result = { status: 200, json };
+  next();
```

Comment:
**Suggestion:** Consider adding error handling for the async function `getMinReject`. If `calcVoteValue.getMinReject(value)` fails, it will throw an unhandled promise rejection.
```suggestion
const getMinReject = async (req, res, next) => {
  const value = validators
    .validate(req.body, validators.user.minRejectSchema, next);
  if (!value) return;

  try {
    const json = await calcVoteValue.getMinReject(value);
    res.result = { status: 200, json };
  } catch (error) {
    res.result = { status: 500, json: { error: 'Failed to calculate minimum rejection value' } };
  }

  next();
};
```